### DISCUSSION TITLE
Confirming schema of the part_config table

### DISCUSSION THREAD

**Discussion by neetam15** [pgpartman/pg_partman#414] [Q&A] [answered]
I created the partman extension on a Postgres 12 docker instance, the version of partman is 4.5.0. I have not created the partman schema yet and I see that the part_config table is present in the public schema. _(partman schema is created later and I don't see any tables in it.)_ Eventually after all the setup, my run_maintenance calls successfully achieve the required partitions. 

On an RDS server however, I am trying using partman 4.5.1. On creation of the extension, I don't see the part_config table in the public schema. 

I am using `CREATE EXTENSION pg_partman;` (without specifying the schema). Should I be creating the schema first and then creating the extension like below?
```
CREATE SCHEMA partman;
CREATE EXTENSION pg_partman WITH SCHEMA partman;
```

**Comment by keithf4** [comment, marked as answer]
Installing an extension into a specific schema is completely optional. I recommend it for pg_partman because most of its functions are pretty self-contained and its easier to control access to extension objects if they are in a dedicated schema. However if you want to install it to the public/default schema for your system, it will still work perfectly fine. If you're not seeing it in the public schema, you may have another schema before it in your search_path. Run the following to see:
```
show search_path;
```

If you want to install it to a specific schema, that schema will have to exist first. If it's already installed and you need to move it to a specific schema, you will have to remove and reinstall it. This is not a relocatable extension.
